src/main/java/com/github/fge/jsonpatch/DualPathOperation.java

```diff
@@ -72,6 +72,10 @@ public final void serializeWithType(final JsonGenerator jgen,
         serialize(jgen, provider);
     }
 
+    public JsonPointer getFrom() {
```

Comment:
Why not `final`? Here and elsewhere.

Reply:
Should have had this in, implemented now